### ISSUE TITLE
[Feature] Add docker image

### ISSUE THREAD

**Issue by ToshY** [bastien70/dbsaver#59]
Hey @bastien70 & @jmsche 👋 

I like the idea and setup of this bundle, so I would like to know if you are open to having a docker image for it?
Even though I'm kind of new to making images myself, I've already started working on it, so if you are interested I'd love to create a PR and hear your feedback.

**Comment by bastien70** [comment]
Hi ! Yes of course it's a good idea :)

If you are familiar with the French language (or not), a youtuber made a video in which he dockerized a symfony application. I don't know if it meets your needs : https://www.youtube.com/watch?v=KFWnB5hW6j8

**Comment by ToshY** [comment]
Hey @bastien70 

Thanks for the additional information. **_Je comprends un peu le français_**, so I was able to follow through. I was almost done with my Dockerfile, which is fairly similar to the one that youtuber made. But I noticed he's missing out on some best practices for creating docker images, so instead I'm taking [dunglas/symfony-docker](https://github.com/dunglas/symfony-docker) as a better guideline.
